stable/aws-pod-identity-webhook/values.yaml

```diff
@@ -0,0 +1,24 @@
+tls_secret_name: pod-identity-webhook
+annotation_prefix: eks.amazonaws.com
+token_audience: sts.amazonaws.com
```

Comment:
For above values, please use camelCase instead of underscores in order to match the style of the other charts and the values below.